### PR TITLE
Control-br update

### PR DESCRIPTION
**Pull request by lututui**
* Syncs control-br folder with main control folder
* Removes duplicate explanation in pickupitems.txt and shop.txt

### CODE REVIEW

**Comment by Cozzie** [comment]
@itsrachelfish Please review. Small changes.

**Comment by flacrasec** [comment]
Wow this is quite a cleaning job lututui thank you again! =) I was not using the translated control because it was outdated but now maybe i return to the brazilian version ;)

**Comment by lucasoli** [comment]
Thanks lutuitui, you did an awesome job!

**Comment by itsrachelfish** [comment]
Reviewed the changes, this all looks good. I feel like I'm learning Portuguese :smile:

File: control/pickupitems.txt [hunk 1]
```diff
@@ -7,10 +7,6 @@
 #		 1 to pickup the item,
 #		 2 to take the item as fast as it can
 #
-#<item name> - name of item (not case sensitive)
-#
-#<flag> - 1 to pickup the item, 0 to leave it alone
-#
```
Comment (gasaraki00) [inline control/pickupitems.txt]: Syncs control-br folder with main control folder
Removes duplicate explanation in pickupitems.txt and shop.txt

**Pull request merged**